lib/src/util/messages.rs

```diff
@@ -1,3 +1,7 @@
+//! Utility traits that extends [`telegram_bot_raw::types::message`]
```

Comment:
I had the same inclination here: https://github.com/telegram-rs/telegram-bot/pull/145#issuecomment-538757120

But after some thoughts I thought that maybe it would be better to let `telegram-bot-raw` only have essential types so if someone builds on top of it, they won't have to bring in types that are utility only. Do you think it's more valuable to have that extension for everyone who would be only `telegram-bot-raw` users?